client/ayon_core/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -15,8 +14,11 @@
     set_instances,
     remove_instance
 )
-from ayon_core.hosts.substancepainter.api.lib import get_export_presets
+from ayon_core.hosts.substancepainter.api.lib import (
+    get_export_presets
+)
```

Comment:
Style change seems redundant?
```suggestion
from ayon_core.hosts.substancepainter.api.lib import get_export_presets
```